Wrapping/Generators/CMakeLists.txt

```diff
@@ -58,7 +58,102 @@ endmacro()
 macro(itk_end_wrap_submodule_all_generators module)
   itk_wrap_modules_set_prefix()
   if(${module_prefix}_WRAP_CASTXML)
-    itk_end_wrap_submodule_castxml("${module}")
+   # write the wrap_*.cxx file
+   #
+   # Global vars used: WRAPPER_INCLUDE_FILES WRAPPER_MODULE_NAME and WRAPPER_TYPEDEFS
+   # Global vars modified: none
+
+   # Create the cxx file which will be given to castxml.
+   set(cxx_file "${WRAPPER_LIBRARY_OUTPUT_DIR}/${module}.cxx")
```

Comment:
Commit message does not match this addition.

Reply:
Fixed and squashed several commit together.